### ISSUE TITLE
[BUG] No matching distribution found for nni==0.5.2.1 on Windows DSVM

### ISSUE THREAD

**Issue by roalexan** [Microsoft/Recommenders#747]
### Description
Seeing an error when installing nni on Windows. This is the error that I see when setting up the environment:

DEBUG menuinst_win32:__init__(196): Menu: name: 'Anaconda${PY_VER} ${PLATFORM}', prefix: 'C:\Anaconda\envs\nightly_reco_base', env_name: 'nightly_reco_base', mode: 'system', used_mode: 'system'
DEBUG menuinst_win32:create(320): Shortcut cmd is C:\Anaconda\python.exe, args are ['C:\\Anaconda\\cwp.py', 'C:\\Anaconda\\envs\\nightly_reco_base', 'C:\\Anaconda\\envs\\nightly_reco_base\\python.exe', 'C:\\Anaconda\\envs\\nightly_reco_base\\Scripts\\jupyter-notebook-script.py', '"%USERPROFILE%/"']
  Could not find a version that satisfies the requirement nni==0.5.2.1 (from -r C:\Users\recocat\agent\_work\1\s\condaenv.ieyjoeec.requirements.txt (line 11)) (from versions: 0.3.2, 0.3.3, 0.3.4)
No matching distribution found for nni==0.5.2.1 (from -r C:\Users\recocat\agent\_work\1\s\condaenv.ieyjoeec.requirements.txt (line 11))

CondaValueError: pip returned an error

### In which platform does it happen?
Windows DSVM

### How do we replicate the issue?
Follow the setup instructions for testing on Windows. Creating the reco_base python environment will cause the error.

### Expected behavior (i.e. solution)
no error. not sure about solution - we'll probably have to conditionally disable this somehow on Windows.

**Comment by gramhagen** [comment]
it seems as though they are not supporting windows past version 0.3.4 (at least from what is availabily on PyPI)

i'm not sure if the repo is cloned if it can still be built on windows, but this introduces a challenge both on the yaml file generation side and the unit testing side. Should we check the OS system and remove nni / nni tests?

**Comment by roalexan** [comment]
In addition to not running the nni tests on Windows, we'll have to see how to create the python environment, minus this one package. Or maybe use the latest installable nni package on either environment and see if the tests run.

**Comment by gramhagen** [comment]
yeah, it's feasible to remove this package in the generate_conda_file script if it detects a windows system, but it seems less than ideal. Let's hold until @anargyri can weigh in.

@scarlett2018 do you know if windows support has intentionally been removed from the latest PyPI versions?

**Comment by gramhagen** [comment]
oh, i see the documentation describes that too, only linux and mac: https://nni.readthedocs.io/en/latest/Installation.html

**Comment by scarlett2018** [comment]
> oh, i see the documentation describes that too, only linux and mac: https://nni.readthedocs.io/en/latest/Installation.html

@gramhagen - right, Windows support is still on the way. we are planning to support Windows in the upcoming April release.  https://github.com/Microsoft/nni/issues/944

**Comment by gramhagen** [comment]
ok, thanks! so I think the best approach for now is to skip nni install / testing if the platform is windows. I've started a branch for that, will push to PR later today.
